src/unstract/sdk/ocr.py

```diff
@@ -0,0 +1,41 @@
+from abc import ABCMeta
+
+from unstract.adapters.constants import Common
+from unstract.adapters.ocr import adapters
+from unstract.adapters.ocr.ocr_adapter import OCRAdapter
+
+from unstract.sdk.adapters import ToolAdapter
+from unstract.sdk.constants import LogLevel
+from unstract.sdk.tool.base import BaseTool
+
+
+class OCR(metaclass=ABCMeta):
+    def __init__(self, tool: BaseTool):
+        self.tool = tool
+        self.ocr_adapters = adapters
+
+    # @abstractmethod
```

Comment:
Remove if not necessary

Reply:
Lines 14 and 15 are required @johnyrahul

Reply:
Talking about line 18->20

Reply:
Done